drivers/SmartThings/zigbee-button/fingerprints.yml

```diff
@@ -1,6 +1,6 @@
 zigbeeManufacturer:
   - id: "HEIMAN/SOS-EM"
-    deviceLabel: HEIMAN Button
+    deviceLabel: 海曼智能紧急按钮
```

Comment:
Just to confirm - these device label changes are going to be reverted back to the english names, right?